builder/files/bor.service

```diff
@@ -6,7 +6,7 @@
 [Service]
   Restart=on-failure
   RestartSec=5s
-  ExecStart=/usr/local/bin/bor \
+  ExecStart=/usr/local/bin/bor server \
     --bor-mumbai \
```

Comment:
Do we need to replace this with `--chain=mumbai`?

Reply:
Yeah probably